include/a3/editor_f/Data/Scripts/dikCodes.h

```diff
@@ -0,0 +1,181 @@
+#ifndef DIK_ESCAPE

```

Comment:
What is the point of this file? I didn't catch it being used anywhere

Reply:
> What is the point of this file? I didn't catch it being used anywhere

It was supposed to be added with the keybinds addon